docs/activegraph.rb

```diff
@@ -1,6 +1,6 @@
 # Usage: rails new myapp -m activegraph.rb
 
-gem 'activegraph', '=> 11.1'
+gem 'activegraph', '>= 11.1'
```

Comment:
I generated fresh rails application before and after this change to compare exact file difference after this change, [here is its diff](https://github.com/mrhardikjoshi/myapp/pull/1/files)